salesagility/SuiteCRM
# Acceptance Tests for company module (Sugar Objects)

Task: SCRM-663

This change includes #4044 - Please review and merge this in first.

## Description



This change adds acceptance test to the company sugar object. It fills in all the fields default fields show in the edit view.

I have added a clickSaveButton action in the EditView step. This ensures that the tester can see the edit view save button, and prevents potential test errors in the smaller devices.

There is a known issue with the filter on smaller devices. The Quick filter doesn't display as a link and so the tester cannot click on it.

## Motivation and Context

Tests company module:
- create company module in module builder
- view company module
- create record with company module
- filter and select record
- edit record
- duplicate record
- delete record

## How To Test This

* See [tests/README.md](https://github.com/daniel-samson/SuiteCRM/blob/feature_codeception_basic_module/tests/README.md) and configure your environment appropriately
* Run `composer install`
* Run `codecept run install`
* Run `codecept run tests/acceptance/Core/CompanyModuleCest.php`

## Types of changes


### Final checklist
